packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -114,6 +116,8 @@ export function Drawer({
           {...dialogProps}
           ref={overlayRef}
         >
+          {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+          <div className={styles.resizer} onMouseDown={onMouseDown} />
```

Comment:
we need to consider touch/keyboard access here as well. keyboard access we can handle in a follow up PR, but i think touch access would be useful to have as part of this PR as you're even more likely to want to resize the drawer on a touch device. think it should be a case of duplicating the `onMouseDown` logic but with `onTouchStart`, `touchmove` and `touchend`

Reply:
@ashharrison90 added touch support